src/main/java/by/issoft/training/pages/Page.java

```diff
@@ -0,0 +1,49 @@
+package by.issoft.training.pages;
+
+import by.issoft.training.driver.Driver;
+import by.issoft.training.util.Random;
+import by.issoft.training.util.Waiter;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Page {
+
+	protected WebDriver driver;
+	protected static String subjectText = Random.randomizeSubject("Subject+");
+
+	public Page() {
+		driver = Driver.getDriver();
+		PageFactory.initElements(driver, this);
+	}
+
+	public boolean isPageLoadInTime(WebElement element1, WebElement element2) {
+		if (Waiter.getWebDriverWait().until(ExpectedConditions.visibilityOf(element1)).isDisplayed()
```

Comment:
It makes sense to move this expression in private method

Reply:
Also there is no needs to return true or false. Just return expression/method, f.e.
return isSomethingDisplayed();